lib/game_play.rb

```diff
@@ -0,0 +1,120 @@
+require './lib/player'
+require './lib/board'
+require './lib/cell'
+require 'pry'
+
+class GamePlay
+
+  attr_reader :human,
+              :computer
+
+  def initialize(human, computer)
+    @human = human
+    @computer = computer
+    @already_fired_upon_coordinates_human = []
+    @already_fired_upon_coordinates_computer = []
+  end
+
+
+    def display_boards
+      sleep(1)
+      puts "=============COMPUTER BOARD============="
+      puts @computer.board.render
+      puts "==============PLAYER BOARD=============="
+      puts @human.board.render(true)
+    end
+
+    def player_chooses_coordinate
+      puts "Enter the coordinate for your shot: \n"
+      input = gets.upcase.chomp
+
+
+      loop do
+
+        if @computer.board.valid_coordinate?(input) && @already_fired_upon_coordinates_human.include?(input) == false
```

Comment:
let's do !already_fired_...etc instead of == false

Reply:
Yeah, agreed. I really appreciated when Brian pointed that out to us earlier today--I'd say ==false is more readable, but !@already... is more elegant. Ultimately, it's nice to not have to write as much. Just need to make sure we keep track of that !.